index.js

```diff
@@ -0,0 +1,36 @@
+module.exports = App
+
+const jsonwebtoken = require('jsonwebtoken')
+const request = require('@octokit/rest/lib/request')
+
+function App ({id, privateKey}) {
+  const api = {
+    appAuth () {
+      const now = Math.floor(Date.now() / 1000)
+      const payload = {
+        iat: now, // Issued at time
+        exp: now + 60, // JWT expiration time
+        iss: id
+      }
+      const token = jsonwebtoken.sign(payload, privateKey, {algorithm: 'RS256'})
+      return token
+    },
+
+    async requestToken ({installationId}) {
+      const res = await request({
+        // POST /app/installations/:installation_id/access_tokens
+        method: 'POST',
+        url: `/app/installations/:installation_id/access_tokens`,
+        installation_id: installationId,
+        headers: {
+          accept: 'application/vnd.github.machine-man-preview+json'
+        },
+        // TODO: store the appAuth() if it's been elss than 60 minutes
```

Comment:
```suggestion
        // TODO: store the appAuth() if it's been less than 60 minutes
```

^^ Just wanted to try the new suggestion feature!